src/bindings/lua/wreck.lua

```diff
@@ -43,6 +43,7 @@ local default_opts = {
     ['help']    = { char = 'h'  },
     ['verbose'] = { char = 'v'  },
     ['ntasks']  = { char = 'n', arg = "N" },
+    ['ngpus']  = { char = 'g', arg = "g" },
```

Comment:
Unfortunately the options structure in the Lua code is not auto-documenting like optparse. Documentation of the `--ngpus` option should also be added to `wreck:usage()`.

Also consider adding to the wreckrun and submit manpages, but I think unfortunately those need a full editing pass so we can save it for later if you'd like.